pkg/netset/ipblock.go

```diff
@@ -156,6 +164,18 @@ func (b *IPBlock) Split() []*IPBlock {
 	return res
 }
 
+// SplitToCidrs returns a set of Cidrs
+func (b *IPBlock) SplitToCidrs() []*IPBlock {
+	cidrs := make([]*IPBlock, 0)
+	for _, ipRange := range b.ipRange.Intervals() {
+		for _, cidrString := range intervalToCidrList(ipRange) {
+			ipblock, _ := IPBlockFromCidrOrAddress(cidrString)
```

Comment:
```suggestion
			ipblock, _ := IPBlockFromCidr(cidrString)
```
Save the check for the CIDR separator